src/components/all-jobs.tsx

```diff
@@ -30,32 +33,64 @@ const AllJobs = async ({ searchParams }: PaginatorProps) => {
       {jobs.additional.jobs.length > 0 ? (
         jobs.additional?.jobs.map((job) => {
           return (
-            <Link key={job.id} href={`/jobs/${job.id}`}>
-              <div
-                className="w-[94%] mx-auto flex flex-col items-start gap-4 rounded-lg border p-3 text-left text-sm transition-all hover:bg-accent"
-                key={job.id}
-              >
-                <div className="flex w-full flex-col gap-2">
-                  <p className="font-semibold">{job.title}</p>
-                  <p className="text-xs font-medium">{job.companyName}</p>
+            <Link
+              key={job.id}
+              href={`/jobs/${job.id}`}
+              className="sm:text-sm text-xs text-slate-500 dark:text-slate-400 font-medium flex flex-col border p-6 bg-slate-100 gap-4 dark:bg-slate-900 hover:bg-slate-200 dark:hover:bg-slate-800 rounded-lg "
+            >
+              <div className="flex w-full gap-3">
+                <div>
+                  {/* Todo : replace with job.companyLogo */}
+                  <Image
+                    alt="company_logo"
+                    src={'/spotify.png'}
```

Comment:
why the src is harcoded here ? while seeing put the image link / logo

Reply:
for now, we don't have the company image in the seed file, so I hardcoded it